src/styles/components/artifacts-section/styles.less

```diff
@@ -0,0 +1,3 @@
+.artifacts-section {
+  margin-top: 18px;
```

Comment:
As part of a separate PR, I think we need to go back and consistently apply margins to different sections of the form. I created a JIRA for it here: https://mesosphere.atlassian.net/browse/DCOS-11803

Would you mind including the ticket number and a `TODO` in this file?